### PR TITLE
Add trust_remote_code arg to get_config

### PR DESCRIPTION
**Pull request by WoosukKwon**
This PR adds the `trust_remote_code` argument to the `get_config` function.

### CODE REVIEW

File: vllm/transformers_utils/config.py [hunk 1]
```diff
@@ -7,8 +7,19 @@
...
+            model, trust_remote_code=trust_remote_code)
+    except ValueError as e:
+        if (not trust_remote_code and
+                "requires you to execute the configuration file" in str(e)):
+            err_msg = (
+                "Failed to load the model config. If the model is a custom "
+                "model not yet available in the HuggingFace transformers "
+                "library, consider setting `trust_remote_code=True` in LLM "
+                "or using the `--trust-remote-code` flag in the CLI.")
+            raise RuntimeError(err_msg) from e
```
Comment (zhuohan123) [inline vllm/transformers_utils/config.py:22]: Please add an else branch to make sure we do not miss any errors:
```suggestion
        if (not trust_remote_code and
                "requires you to execute the configuration file" in str(e)):
            err_msg = (
                "Failed to load the model config. If the model is a custom "
                "model not yet available in the HuggingFace transformers "
                "library, consider setting `trust_remote_code=True` in LLM "
                "or using the `--trust-remote-code` flag in the CLI.")
            raise RuntimeError(err_msg) from e
        else:
            raise e
```

**Review by zhuohan123 — CHANGES_REQUESTED** [review]
Thanks for fixing this! Left some comments.

File: vllm/transformers_utils/config.py (hunk 1 above)
Comment (WoosukKwon) [reply vllm/transformers_utils/config.py:22]: Nice catch! Just added it. Thanks.

**Review by zhuohan123 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**